crates/feature_flags/src/feature_flags.rs

```diff
@@ -74,6 +74,15 @@ impl FeatureFlag for AutoCommand {
     }
 }
 
+pub struct SearchCommand {}
+impl FeatureFlag for SearchCommand {
+    const NAME: &'static str = "search-command";
```

Comment:
We already have a flag here that we can reuse: https://github.com/zed-industries/zed/blob/7c72929f0ba3ade5fde3e7b090dcd2685ae243e3/crates/assistant/src/slash_command/search_command.rs#L20-L24